src/opera/pge/disp_s1/schema/disp_s1_sas_schema.yaml

```diff
@@ -32,6 +32,9 @@ dynamic_ancillary_file_group:
     # troposphere corrections are skipped.
     troposphere_files: list(str(), required=False, min=0)
 
+    # For STATIC workflow ONLY
```

Comment:
A nice-to-have down the road might be an extra pre-processor input validation check that makes sure this field is only provided when `product_type == DISP_S1_STATIC`

Reply:
That's kind of why my initial approach was to define a separate workflow for static layers since it and baseline have differing sets of required keys

Reply:
@collinss-jpl I added a function to the DISP-S1[-STATIC] preprocessors to check for these options